packages/sdks-tests/src/snippet-tests/editable-regions.spec.ts

```diff
@@ -45,4 +45,19 @@ test.describe('Editable regions in custom components', () => {
     const secondText = await columns.nth(1).textContent();
     expect(secondText?.trim().toLowerCase()).toBe('column 2 text');
   });
+
+  test.describe('Remix gen1 editable regions text validation', () => {
```

Comment:
why does remix gen1= have its own special test here? why aren't we enabling the existing tests in this spec for remix?